### PR TITLE
bootstrap

### PR DESCRIPTION
**Pull request by RichardBongani**

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Richard!

### Nice job! Project looks! :+1: 

But first of all let's fix the next problems and we can move on.

* Fix problems with linter now it`s not working :(

* Kindly recommended checking HTML markup with this [validator](https://validator.w3.org/#validate_by_upload)

Now it shows a lot of ERRORs in your project

Screenshot from 2019-11-28 21-27-38


* Your README.md file should be descriptive, I hope this short [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) can help you to make this file.

Now you readme file without any descriptions

reedme

* Please check responsiveness of your project, as I can see all parts have problems :(

For example :

3humu2



-----------------------------------------

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!


### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

**Review by Anna-Myzukina — COMMENTED** [review]
The project needs changes.

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
Linter does not work correctly. The HTML file has errors, readme file is not descriptive. The project isn`t responsive.

**Comment by RichardBongani** [comment]
> The project needs changes.

Awesome, thank you Anna. Working on it.

**Comment by Anna-Myzukina** [comment]
### Hi Richard!

### Great job!  :+1: 

But your project still needs changes

* Please check problems with linter now it shows errors, kindly recommended fix it :(
Follow this link to see errors: [Details](https://github.com/RichardBongani/Newsweek.com/pull/1/checks?check_run_id=354981141)


linter

* Kindly recommended checking HTML markup with this [validator](https://validator.w3.org/#validate_by_upload)

screen
It shows a lot of ERRORs in your project

validator

screen
* Please check responsiveness of your project, as I can see all parts have problems :(

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1146 @@
...
+            </div>
+            <div class="col-md-6 col-6 news divider">
+              <p class="elizabeth-warren pi"><a href="#" class="link-official">Elizabeth Warren Could Withhold U.S. Aid to
+                  Israel Over Settlements</a></p>
+            </div>
+          </div>
+          <div id="sign-up">
+            <p class="newsletter">SIGN UP FOR OUR<br> NEWSLETTER</p>
+            <img class="news-img" src="images/news.png" alt="my_newslet">
+            <button type="button" name="sign-up" class="sign-newsletter">SIGN UP ></button>
```
Comment (Anna-Myzukina) [inline index.html]: Line number 472: Special characters must be escaped : [ > ].

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1146 @@
...
+            <p class="preferences"><a href="#" class="pref-link">Update your preferences »</a></p>
+          </div>
+        </section>
+        </section>
+      </div>
+    </div>
```
Comment (Anna-Myzukina) [inline index.html:662]: Line number 478: Tag must be paired, no start tag: [ </div> ]

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1146 @@
...
+        <h6 class="magazine-title">IN THE MAGAZINE</h6>
+        </div>
+        <hr class="my_mag_line">
+        <div class="row container-fluid">
+          <div class="col-md-4">
+            <img class="pence"
+              src="https://d.newsweek.com/en/full/1536427/october-25-2019.jpg?w=360&amp;h=480&amp;q=90&amp;f=1f835f26d4a271107746fae6b8b1d8e1"
+              alt="pence">
+            <p class="see-features"><a href="#" class="header-link">SEE ALL FEATURES ></a></p>
+          </div>
```
Comment (Anna-Myzukina) [inline index.html]: Line number 493: Special characters must be escaped : [ > ].

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1146 @@
...
+              alt="cover-us">
+            <p class="us">COVER</p>
+            <p class="party-chatter"><a href="#" class="pic-link">A Pence Presidency Is No Longer Just Dinner PartyChatter—Is He Up to It?</a></p>
+          </div>
+          <div class="col-md-3 cover1">
+            <img class="cover-us"
+              src="https://d.newsweek.com/en/full/1535403/fe-recession-banner-526940218.jpg?w=466&amp;h=311&amp;q=90&amp;f=72944842440003262c09880bca439697"
+              alt="features">
+            <p class="feat">FEATURES</p>
+            <P class="recession"><a href="#" class="pic-link">How to Survive a Recession: 12 Steps You Should Take to
```
Comment (Anna-Myzukina) [inline index.html]: Line number 507: The html element name of [ P ] must be in lowercase.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1146 @@
...
+                Protect Your Money</a></p>
+          </div>
+          <div class="col-md-2 cover1">
+            <img class="cover-us1  cover-us"
+              src="https://d.newsweek.com/en/full/1529202/mattis-marshall-02.jpg?w=466&amp;h=311&amp;q=90&amp;f=ccadd6e26870d307c209dbda06d68aa5"
+              alt="periscope">
+            <p class="periscope">PERISCOPE</p>
+            <P class="jim"><a href="#" class="pic-link">Does Jim Mattis Have a Duty to Criticize Donald Trump?</a></p>
```
Comment (Anna-Myzukina) [inline index.html]: Line number 515: The html element name of [ P ] must be in lowercase.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1146 @@
...
+                        alt="best-seller">
+                      </div>
+                        <div class="mig12">
+                      <img class="president" src="https://d.newsweek.com/en/full/1536427/october-25-2019.jpg?w=170&amp;" alt="pres">
+                      <div class="col-md-6 col-lg-6 indepth">
+                        <p class="school-safety all1">In-depth Stories You Don't See Elsewhere About Topics You Want to
+                          Read!</p>
+                        <p class="all-access all1">All Access + Weekly Delivery.<br>
+                          Choose A Membership That's Perfect for You!</p>
+                        <button type="button" name="subscribe" class="my_sub all1">SUBSCRIBE ></button>
```
Comment (Anna-Myzukina) [inline index.html]: Line number 1050: Special characters must be escaped : [ > ].

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1146 @@
...
+                      <h6 class="news_l">NEWSLETTER</h6>
+                      <img class="chrissy1"
+                        src="https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&h=265&q=90&f=742bce9a6f72dd8f19ba2e7fd5db06e5"
+                        alt="nba_celebration">
+                    </div>
+                    <div class="col-lg-6 col-md-6 daily_cont">
+                      <h4 class="dai_head"><b>Daily</b></h4>
+                      <p class="daily">Take back your inbox. Sign up for our newsletter and start making sense of the
+                        world around you.</p>
+                      <button class="sign-btn" type="button" name="btn">SIGN UP ></button>
```
Comment (Anna-Myzukina) [inline index.html]: Line number 1070: Special characters must be escaped : [ > ].

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project still needs changes

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project is not responsive, pull request requires description.

**Comment by pelzolga123** [comment]
## Hi Richard!  
### You did a good job fixing linter errors :+1: , but you didn't make all changes required previously.
 * [ ] Tablet mode of your project should look like original one. [Link to page.](https://www.newsweek.com/) Right now it looks like this:
image
 
Original:
image

* [ ] Mobile mode is not looking like an original one too:
image

Original:
image

* [ ] Don't forget to add description for pull request of  changes you implemented, so that your project could look more professional. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Requires changes.

**Comment by pelzolga123** [comment]
### You made an attempt, but still all changes, required previously need to be done. 
* You tried to fix tablet mode responsiveness, but it still looks not good:
image

* You also have stickler error now, that needs fixing.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work so far @RichardBongani! 
Your site is going to be really great :rocket:. Please look deeply into your mobile styling, especially the tablet mode, the page isn't responding like it should to different screen sizes.

When you fix this, please submit for a re-review!

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Hello @RichardBongani!
Your responsiveness is still a bit jagged, please look into it thoroughly. Compare what happens at each breakpoint on the original page to what happens on yours. A good way to do this might be to move sequentially through each piece/element on the page and make all appropriate change and improvements.

Try to settle down and make sure everything is in order so you don't keep going back and forth on reviews. All the best! :rocket:

**Comment by izaiasneto4** [comment]
### Hi @RichardBongani :wave: 

Nice work implementing some of the features, we're almost there :muscle: Let's make a list of the things that o need to finish before re-submitting for code review :arrow_down: 

- [ ]  Your header is still not properly adapted for the tablet/cellphone views, see the screenshot below

image
Original website

image
Your clone
- [ ] Some of the images look too out of proportion, giving the website a not so great look. Let's fix those to make your project look better
image

- [ ] When resizing your project I've noticed that some sections overlap each other, let's try to fix those as well  :wink: 
image

- [ ] Your footer is not in the right proportion for the tablet version (some of the information is overlapping and some is missing)
image

- [ ] In the mobile and tablet version, your project has this big blank spaces, let's fix those as well :+1: 
image

- [ ] The "subscribe section" is out proportion and overlapping the other section in the mobile version, see screenshot
image

**Please, when finished with the request, re-submit your project for a new code review**. :pray: 

**TSE: Izaias Neto**

**Review by imhta — APPROVED** [review]
# Project approved 🎉 

### Hello  @RichardBongani  , 👋 You have done a great job in this project 👏 



> [Optional] fix horizontal scrolling and logo resolution, if you do clone will look great 
----
### Happy coding 😄 
----
**Pon Muthu Selvam N**
__nponmuthuselvam@gmail.com__

**Pull request merged**